pygin/pygin/src/plugin_module.cpp

```diff
@@ -166,7 +166,27 @@ void plugin_module::FreeFindDataW(const FreeFindDataInfo* Info)
 
 intptr_t plugin_module::GetFilesW(GetFilesInfo* Info)
 {
-	return 0;
+	if (Info->StructSize < sizeof(*Info))
+		return 0;
+
+	auto& Context = *reinterpret_cast<context*>(Info->hPanel);
+	auto PanelItems = py::list(Info->ItemsNumber);
+	for (int i = 0; i < Info->ItemsNumber; ++i)
+	{
+		auto const& Item = Info->PanelItem + i;
```

Comment:
`Info->PanelItem[i]`?

Reply:
Fixed in 25bec961a4332cba0ae5649d2dd13af0bff6261c.